lib/elixir/lib/module.ex

```diff
@@ -1089,10 +1089,10 @@ defmodule Module do
         "could not call #{fun} on module #{inspect module} because it was already compiled"
   end
 
-  defp warn_if_redefining_attribute(nil, _table, _key, _value), do: false
-  defp warn_if_redefining_attribute(stack, table, key, value) do
+  defp warn_if_redefining_attribute(nil, _table, _key), do: false
+  defp warn_if_redefining_attribute(stack, table, key) do
     case :ets.lookup(table, key) do
-      [{_, v}] when not v in [nil, false, value] ->
+      [{_, val}] when val != nil and elem(val, 1) != false ->
```

Comment:
There is for sure typedoc too. they are all listed somewhere in this file or kernel.